Mathlib/Algebra/Group/Subgroup/Map.lean

```diff
@@ -491,6 +491,11 @@ additive subgroup to itself."]
 def subgroupComap (f : G →* G') (H' : Subgroup G') : H'.comap f →* H' :=
   f.submonoidComap H'.toSubmonoid
 
+@[to_additive]
+lemma subgroupComap_surjective_of_surjective (f : G →* G') (H' : Subgroup G') (hf : Surjective f) :
```

Comment:
```suggestion
lemma subgroupComap_surjective_of_surjective {f : G →* G'} (H' : Subgroup G') (hf : Surjective f) :
```
And similarly in the other cases.

Reply:
The `subgroupComap_surjective_of_surjective` is in the namespace `MonoidHom`, so making `f : G →* G'` explicit will enable the use of dot notation.